framework/wazuh/rbac/orm.py

```diff
@@ -2721,6 +3064,23 @@ def get_database_version(self, database: str) -> str:
         """
         return str(self.sessions[database].execute(text("pragma user_version")).first()[0])
 
+    def check_if_reserved_id(self, from_id: str | None, to_id: str | None):
+        """Checks if the ids are reserved ones
+
+        Parameters
+        ----------
+        from_id : str | None
+            ID which the resources will be migrated from.
+        to_id : str | None
+            ID which the resources will be migrated to.
+
+        Returns
+        -------
+        bool
+            Condition result
+        """
+        return from_id == WAZUH_USER_ID and to_id == WAZUH_WUI_USER_ID
```

Comment:
```suggestion
    def check_if_reserved_id(self, from_id: Optional[str], to_id: Optional[str]):
        """Checks if the ids are reserved ones.

        Parameters
        ----------
        from_id : Optional[str]
            ID which the resources will be migrated from.
        to_id : Optional[str]
            ID which the resources will be migrated to.

        Returns
        -------
        bool
            Condition result.
        """
        return from_id == WAZUH_USER_ID and to_id == WAZUH_WUI_USER_ID
```

Also, this method could be a function outside the class or a static method.

Reply:
Changed in 0555ba55fe8fe62b03567491c022bcdc7be88237.